### PR TITLE
UI for adding subtitles

### PR DESCRIPTION
**Pull request by drillprop**
#2490 
~I'm still waiting for two icons in the context menu(cc and download). 👇🏻  Once Kuba adds them I update my code as well.~

### CODE REVIEW

**Review by rafalpawlow — CHANGES_REQUESTED** [review]
I think we should update description here
Zrzut ekranu 2022-08-9 o 10 43 08

**Comment by drillprop** [comment]
Draft for now. I need to do some changes regarding closed captions

**Comment by drillprop** [comment]
After some discussions with Kuba, we decided to make some tweaks to the flow. All the changes are listed here: https://github.com/Joystream/atlas/issues/3101.

File: packages/atlas/src/components/_overlays/ContextMenu/ContextMenu.tsx [hunk 1]
```diff
@@ -6,22 +6,26 @@ import { ListItemProps, ListItemSizes } from '@/components/ListItem'
...
+  size?: ListItemSizes
+  customWidth?: number
+} & Omit<PopoverProps, 'content' | 'instanceRef'>
 
 export const ContextMenu: FC<ContextMenuProps> = ({
   children,
   items,
   scrollable = false,
   size = 'medium',
+  customWidth = 192,
```
Comment (rafalpawlow) [inline packages/atlas/src/components/_overlays/ContextMenu/ContextMenu.tsx]: I think we don't need this anymore

**Review by rafalpawlow — CHANGES_REQUESTED** [review]

**Review by rafalpawlow — APPROVED** [review]

File: packages/atlas/src/components/_inputs/ComboBox/ComboBox.tsx [hunk 2]
```diff
@@ -70,15 +70,10 @@ export const ComboBox = <T extends unknown>(props: ComboBoxProps<T>) => {
       }
     },
     onInputValueChange: ({ inputValue }) => {
-      if (!inputValue) {
```
Comment (kdembler) [inline packages/atlas/src/components/_inputs/ComboBox/ComboBox.tsx:73]: After changes to this file, whitelisting in NFT sale form doesn't work. No requests are fired after I type in member name

File: packages/atlas/src/components/_inputs/SubtitleBox/SubtitleBox.tsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+      nodeStart: <SvgActionTrash />,
+    },
+  ]
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    if (e.currentTarget.files?.[0]?.type !== 'text/vtt') {
+      openUnsuportedFileDialog({
+        title: 'File format unsupported',
+        description:
+          'It looks like you selected a file format which is not supported. Reselect file and choose .webvtt file to proceed.',
```
Comment (kdembler) [inline packages/atlas/src/components/_inputs/SubtitleBox/SubtitleBox.tsx]:
```suggestion
          'It looks like you selected a file format which is not supported. Reselect file and choose .vtt file to proceed.',
```

File: packages/atlas/src/components/_inputs/SubtitlesComboBox/SubtitlesCombobox.tsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  onLanguageDelete,
+  onSubtitlesAdd,
+}) => {
+  const [availableLanguages, setAvailableLanguages] = useState<AvailableLanguage[]>(
+    languages
+      .map((language) => [
+        { language, type: 'subtitles' as const, disabled: false },
+        { language, type: 'closed-captions' as const, disabled: false },
+      ])
+      .flat(1)
```
Comment (kdembler) [inline packages/atlas/src/components/_inputs/SubtitlesComboBox/SubtitlesCombobox.tsx]: I think `1` is a default

**Review by kdembler — CHANGES_REQUESTED** [review]

**Review by kdembler — APPROVED** [review]

**Pull request merged**